view/frontend/templates/order/payments.phtml

```diff
@@ -25,12 +25,29 @@
                     <td data-th="<?= $block->escapeHtml(__('Recurring Payment Profile')) ?>" class="col total"><?= /* @noEscape */ $block->escapeHtml($recurringPayment->getName()) ?></td>
                     <td data-th="<?= $block->escapeHtml(__('Grand total')) ?>" class="col status"><?= $block->escapeHtml($block->getCurrentCurrency()).$block->escapeHtml($recurringPayment->getBaseGrandTotal()) ?></td>
                     <td data-th="<?= $block->escapeHtml(__('Actions')) ?>" class="col actions">
-                        <a href="<?= $block->escapeUrl($block->getViewUrl($recurringPayment)) ?>" class="action view">
-                            <span><?= $block->escapeHtml(__('View Order')) ?></span>
-                        </a>
-                        <a href="<?= $block->escapeHtml($block->getStopPaymentUrl($recurringPayment)) ?>" class="action order stop-payment-confirmation">
-                            <span ><?= $block->escapeHtml(__('Stop subscription')) ?></span>
-                        </a>
+                        <div class="customer-welcome">
```

Comment:
Consider changing the class name. It is not reflective to what the html is ment for.